commands/team.go

```diff
@@ -335,21 +336,23 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	if public {
 		privacy = model.TeamOpen
 	}
-
+	var Error error
```

Comment:
```suggestion
	var errors *multierror.Error
```
We generally avoid upper case variable names in Go. In this case we want to collect the errors in a `*multierror.Error`.